### ISSUE TITLE
add a keyboard shortcut to toggle automatic language switching

### ISSUE THREAD

**Issue by fatihyuksek1** [nvaccess/nvda#12504]
### Is your feature request related to a problem? Please describe.
I cannot set a keyboard shortcut in the input gestures dialog of NVDA for toggling automatic language switching
### Describe the solution you'd like
Please add an automatic language switching item to the input gestures dialog so we can create a keyboard shortcut for it
### Additional context
I want this to be included in NVDA because sometimes I encounter webpages that have wrong language tags and NVDA tries to read the page in the wrong language

**Comment by josephsl** [comment]
Hi, I think, for this one, it is really the fault of website authors who specified wrong languages for content. A suitable alternative to creating a dedicated gesture to toggling automatic language switching is configuration profiles and assigning a command to toggle between profiles (you can do this already). Thanks.

From: Fatih Yüksek ***@***.***> 
Sent: Wednesday, June 2, 2021 1:56 PM
To: nvaccess/nvda ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nvaccess/nvda] add a keyboard shortcut to toggle automatic language switching (#12504)

 


Is your feature request related to a problem? Please describe.


I cannot set a keyboard shortcut in the input gestures dialog of NVDA for toggling automatic language switching


Describe the solution you'd like


Please add an automatic language switching item to the input gestures dialog so we can create a keyboard shortcut for it


Describe alternatives you've considered


Additional context


I want this to be included in NVDA because sometimes I encounter webpages that have wrong language tags and NVDA tries to read the page in the wrong language

**Comment by CyrilleB79** [comment]
This issue is a duplicate of #10253.
Please close it and comment there.
Thanks.

**Comment by feerrenrut** [comment]
closing as a duplicate of  #10253
